content/chainguard/chainguard-images/chainguard-registry/pull-through-guides/artifactory/artifactory-packages-pull-through/index.md

```diff
@@ -126,41 +162,37 @@ Once the image is built, run it. Be sure to include the `-it` and `--entrypoint`
 docker run -it --entrypoint /bin/sh ar-interactive
 ```
 
-Now that the container is running, you can check whether it was configured correctly and that you can pull packages from the remote Artifactory repositories.
+Now that the container is running, you can check whether it was configured correctly and you can pull packages from the remote Artifactory repositories.
 
-First check that the `/etc/apk/repositories` file contains the correct lines:
+First check that the `/etc/apk/repositories` file contains the correct URL:
 
-```shell
+```container
 cat /etc/apk/repositories
+```
 ```output
-https://<ARTIFACTORY-USER-PROFILE>:<WOLFI-TOKEN>@<ARTIFACTORY-HOSTNAME>.jfrog.io/artifactory/cg-wolfi/
-https://<ARTIFACTORY-USER-PROFILE>:<EXTRAS-TOKEN>@<ARTIFACTORY-HOSTNAME>.jfrog.io/artifactory/cg-extras/
+https://$ARTIFACTORY_USER_PROFILE:$CG_PRIVATE_TOKEN@$ARTIFACTORY_HOSTNAME.jfrog.io/artifactory/cg-private/
 ```
 
-Then test that you can install a package from these repositories with `apk`. This example installs the `curl` package:
+Then test that you can install a package from these repositories with `apk`. This example installs the `npm` package:
 
-```shell
-apk add curl
-```
-```output
-fetch https://<ARTIFACTORY-USER-PROFILE>:*@<ARTIFACTORY-HOSTNAME>.jfrog.io/artifactory/cg-wolfi/
-fetch https://<ARTIFACTORY-USER-PROFILE>:*@<ARTIFACTORY-HOSTNAME>.jfrog.io/artifactory/cg-extras/
-(1/1) Installing curl (8.11.0-r0)
-Executing glibc-2.40-r3.trigger
-Executing busybox-1.37.0-r0.trigger
-OK: 576 MiB in 77 packages
+```container
+apk add npm
```

Comment:
Not sure npm is a good example since its part of the JS libraries..

Reply:
getting rid of this section, so this is moot